### PR TITLE
Code quality

### PR DESCRIPTION
**Pull request by mmuckley**
General code quality updates. Also adds `black` and `flake8` to CircleCI. This is hopefully the second-to-last item for Issue #84. I'll be running some tests with this to generate new leaderboard submissions.

### CODE REVIEW

**Comment by anuroopsriram** [comment]
Looks great. 
Since you added black and flake to `circleci/config.yml`, you should tell people to run the same commands before committing  in the documentation. Otherwise, you'll keep getting a lot of build failures.
You can also add pre-commit hooks to run them automatically every time a user commits something.

**Review by anuroopsriram — APPROVED** [review]

**Comment by mmuckley** [comment]
Okay I just modified `CONTRIBUTING.md` with the relevant information. For the moment we can have people run the linters and formatters themselves.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -27,10 +32,29 @@ disclosure of security bugs. In those cases, please go through the process
...
+`black` and `flake8`.
+
+Configurations for the linters are in the root directory, `.flake8` for `flake8` and 
+`mypy.ini` for `mypy`. We use default configurations for black.
+
+If all of the following commands pass without modifications or failures, your code
+should be ready for CircleCI:
+
+```bash
+black experimental
```
Comment (anuroopsriram) [inline CONTRIBUTING.md:48]: Can we simplify this?
```
black experimental fastmri tests
flake8 experimental fastmri tests
mypy fastmri
```

**Review by anuroopsriram — APPROVED** [review]
Looks good except for a minor comment.

File: CONTRIBUTING.md (hunk 1 above)
Comment (mmuckley) [reply CONTRIBUTING.md]: Sure thing - done in the update.

**Pull request merged**